yaks/README.md

```diff
@@ -684,11 +684,35 @@ explicit.
 
 If the object is not a collection, then lookup happens based on the
 class name, and will traverse up the class hierarchy if no suitable
-mapper is found. So for a `class Widget < Thing`, yaks would look for
-
-* `API::WidgetMapper`
-* `API::ThingMapper`
-* `API::ObjectMapper`
+mapper is found. Take the following
+code:
+```ruby
+module Stuff
+  class Thing ; end
+  class Widget < Thing ; end
+end
+```
+The lookup we'll be done as followed.
+
+* If the `namespace` option is set (to `Mappers` for example):
+ * `Mappers::Stuff::WidgetMapper`
+ * `Mappers::Stuff::ThingMapper`
+ * `Mappers::Stuff::ObjectMapper`
+ * `Mappers::Stuff::BasicObjectMapper`
```

Comment:
You're right that it doesn't make much sense to have `BasicObject` and `Object` in here, maybe better to filter them out